### PR TITLE
Fix #887 doesn't destory the silder instance cause "generates mutiple silder instance" test failed

### LINKED ISSUE #887: Error: doesn't destory the silder instance cause "generates mutiple silder instance" test failed
Hi,

I find this bug will effect on test 'formatter' option properly will be failed.

https://github.com/seiyria/bootstrap-slider/blob/9674de904e28b275269609d1a3e837f39c389a83/test/specs/PublicMethodsSpec.js#L16

The problem is when create a new slider instance the unit test code doesn't destroy it, and keep the old value pass to the 'formatter' option test that cause test failed.

Issues
=============
**NOTE:** Before filing a new issue, please search through the open issues list and check to see if an existing issue already exists that describes your problem. If so, leave a comment within that issue with the checklist items described below.

Please make sure you include the following in your issue report where appropriate:

- [ ] JSFiddle (or an equivalent such as CodePen, Plunker, etc) with the an example demonstrating the bug
- [ ] Instructions to replicate the bug in the Fiddle

### PR DESCRIPTION
**Pull request by xwang345**
Pull Requests
=============
Please accompany all pull requests with the following (where appropriate):

- [ ] unit tests (we use [Jasmine 2.x.x](https://jasmine.github.io/2.2/introduction))
- [ ] JSFiddle (or an equivalent such as CodePen, Plunker, etc) or screenshot/GIF with new feature or bug-fix
- [ ] Link to original Github issue (if this is a bug-fix)
- [ ] documentation updates to README file
- [ ] examples within [/tpl/index.tpl](https://github.com/seiyria/bootstrap-slider/blob/master/tpl/index.tpl) (for new options being added)
- [ ] Passes CI-server checks (runs automated tests, JS source-code linting, etc..). To run these on your local machine, type `grunt test` in your Terminal within the bootstrap-slider repository directory

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,7 +53,7 @@
     "grunt-contrib-clean": "0.6.0",
     "grunt-contrib-connect": "0.5.0",
     "grunt-contrib-jasmine": "1.0.3",
-    "grunt-contrib-jshint": "0.11.3",
+    "grunt-contrib-jshint": "^0.11.3",
     "grunt-contrib-less": "0.7.0",
     "grunt-contrib-uglify": "0.2.4",
     "grunt-contrib-watch": "0.5.3",
```
Comment (jespirit) [inline package.json]: Could you remove any changes to this file. Discard changes to `package.json` and try installing `grunt-cli` globally with `npm install grunt-cli -g`

File: test/specs/PublicMethodsSpec.js [hunk 2]
```diff
@@ -14,14 +14,12 @@ describe("Public Method Tests", function() {
     });
 
     it("generates multiple slider instances from selector", function() {
-
-      $(".makeSlider").slider();
-
```
Comment (jespirit) [inline test/specs/PublicMethodsSpec.js]: Don't delete these lines. Keep them as is.

File: test/specs/PublicMethodsSpec.js [hunk 3]
```diff
@@ -14,14 +14,12 @@ describe("Public Method Tests", function() {
...
       var sliderInstancesExists = $(".makeSlider").siblings().is(".slider");
       expect(sliderInstancesExists).toBeTruthy();
-
```
Comment (jespirit) [inline test/specs/PublicMethodsSpec.js]: nit: Leave empty line here

File: test/specs/PublicMethodsSpec.js [hunk 4]
```diff
@@ -14,14 +14,12 @@ describe("Public Method Tests", function() {
...
       var sliderInstancesCount = $(".makeSlider").siblings(".slider").length;
       expect(sliderInstancesCount).toEqual(2);
+      
+      $('.makeSlider').slider('destroy');
```
Comment (jespirit) [inline test/specs/PublicMethodsSpec.js]: Great catch!

File: package.json [hunk 5]
```diff
@@ -40,7 +40,9 @@
   "license": "MIT",
   "readmeFilename": "README.md",
   "gitHead": "67eb8fe3c26205efc94ae95b3ef37a469eef0b81",
-  "dependencies": {},
+  "dependencies": {
+    "grunt-cli": "^1.3.2"
+  },
```
Comment (rovolution) [inline package.json]: please remove the changes to this file

Comment (xwang345) [reply package.json]: Okay, I'll change it.

**Comment by rovolution** [comment]
wait for @jespirit's final approval

**Pull request merged**